CONTRIBUTING.md

```diff
@@ -90,6 +90,9 @@ This is a set of practices we try to live by when developing KUDO. These are jus
 - Since KUDO is developed in multiple timezones, try to keep the PR open for everyone to be able to see it (~24h, keep in mind public holidays)
 - We prefer squash commits so that all changes from a branch are committed to master as a single commit
 - Before you merge, make sure your commit title and description are meaningful. Github by default will list all the individual PR commits when squashing which are rarely insightful. We aim for a clean and meaningful commit history. 
+- Labels: If your PR includes either **breaking changes** or should get additional attention in the release, add one of these label:
+  - `release/highlight` For a big new feature, an important bug fix, the focus of the current release
+  - `release/breaking-change`  For anything that breaks backwards compatibility and requires users to take special care when upgrading to the new version
```

Comment:
what do you think of adding `release/bugfix` and having a category of bugfixes?

Reply:
I've added it, although I'm a bit hesitant to add too many categories. I'm not sure if we want to categorise all our PRs, it may be a goal at some point. Let's see how we're going to use it.